R/select-tbl.R

```diff
@@ -11,18 +11,20 @@
 #' @param dm A [`dm`] object.
 #' @param ... One or more table names of the tables of the [`dm`] object.
 #' `tidyselect` is supported, see [`dplyr::select()`] for details on the semantics.
+#' @inheritParams vctrs::vec_as_names
 #'
 #' @examples
 #' dm_nycflights13() %>%
 #'   dm_select_tbl(airports, fl = flights)
 #' @export
-dm_select_tbl <- function(dm, ...) {
+dm_select_tbl <- function(dm, ..., repair = "check_unique", quiet = FALSE) {
```

Comment:
I'd rather not add new arguments to this exported function now.

Reply:
Alright, I was wondering. But do you agree with throwing an error if duplicated table names would be produced?

Reply:
Good catch. I adapted our tidyselect wrapper to throw errors early.